schema/project-level/project-schema.json

```diff
@@ -199,6 +199,20 @@
             "$ref": "#/definitions/BudgetBreakdown"
           },
           "uniqueItems": true,
+          "minItems": 1,
+          "deprecated": {
+            "deprecatedVersion": "0.9",
+            "description": "This field is deprecated in favor of `budgetBreakdowns`, to enable the inclusion of multiple different budget breakdowns."
+          }
```

Comment:
I think we only implement a deprecation policy starting with 1.0. So, simpler to just delete it.